addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/internal/messages/RFXComLighting1Message.java

```diff
@@ -175,6 +199,55 @@ public String getDeviceId() {
     }
 
     @Override
+    public Command convertToCommand(RFXComValueSelector valueSelector) throws RFXComException {
+
+        Command OHcommand;
+
+        if (valueSelector.getItemClass() == SwitchItem.class) {
+
+            if (valueSelector == RFXComValueSelector.COMMAND) {
+
+                switch (command) {
+                    case OFF:
+                    case GROUP_OFF:
+                        OHcommand = OnOffType.OFF;
+                        break;
+
+                    case ON:
+                    case GROUP_ON:
+                        OHcommand = OnOffType.ON;
+                        break;
+
+                    case DIM:
+                        OHcommand = IncreaseDecreaseType.DECREASE;
```

Comment:
But are you really sure that this is going to work as expected, you return IncreaseDecreaseType.DECREASE for a ` if (valueSelector.getItemClass() == SwitchItem.class) {` but looking at the source of SwitchItem [[source](https://github.com/eclipse/smarthome/blob/master/bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/items/SwitchItem.java)] only the OnOffType and RefreshType commands are supported.